packages/account/src/Sections/Security/AccountLimits/account-limits.jsx

```diff
@@ -69,9 +69,14 @@ const AccountLimits = ({ account_limits, currency, getLimits, is_fully_authentic
     const [is_loading, setLoading] = React.useState(true);
 
     React.useEffect(() => {
+        return onMount();
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, []);
+
+    const onMount = () => {
         if (is_virtual) setLoading(false);
         else getLimits().then(setLoading(false));
-    }, []);
+    };
```

Comment:
This is confusing, while this may work fine, it is not consistent with the regular way we write `useEffect` hooks. Normally you'd return a clean up function or `void` in the `useEffect` callback, since `void` is allowed this will work (because `onMount` returns `void`), but currently it suggests that a call to `onMount` returns a cleanup effect, which it doesn't.

Also, you can move the `onMount` logic to the `useEffect` callback itself since it's not re-used. You can then add the required dependencies into the dependency array.